docs/guide/usage/ota_updates.md

```diff
@@ -52,7 +52,7 @@ ota:
 *NOTE: there is also a property `update_available` which is deprecated*.
 
 ## Manually check if an update is available
-To check if an update is available for your device send a message to `zigbee2mqtt/bridge/request/device/ota_update/check` with payload `{"id": "deviceID"}` or `deviceID` where deviceID can be the `ieee_address` or `friendly_name` of the device. Example; request: `{"id": "my_remote"}` or `my_remote`, response: `{"data":{"id": "my_remote","updateAvailable":true},"status":"ok"}`.
+To check if an update is available for your device send a message to `zigbee2mqtt/bridge/request/device/ota_update/check` with payload `{"id": "deviceID"}` or `deviceID` where deviceID can be the `ieee_address` or `friendly_name` of the device. Example; request: `{"id": "my_remote"}` or `my_remote`, response: `{"data":{"id": "my_remote","updateAvailable":true},"status":"ok"}`. Depending on the device or manufacturer (ex: Xiaomi) you may need to also push the pairing button on the device after triggering OTA to set the battery device into the OTA state.
```

Comment:
I think this is true for every battery powered (end) device

Reply:
Updated